scripts/listener.ts

```diff
@@ -1,11 +1,51 @@
 import * as yargs from 'yargs';
 
 import { spec as EdgewareSpec } from '@edgeware/node-types';
+import * as CloverSpec from '@clover-network/node-tpye';
```

Comment:
Seems like you don't even need this import since you're defining the spec manually below.